ocs_ci/ocs/ui/base_ui.py

```diff
@@ -955,8 +975,11 @@ def login_ui(console_url=None):
         )
     )
     element.click()
-    if default_console:
+    if default_console and username is None:
```

Comment:
can you change ` if default_console is None and username is None`

Reply:
done